target/linux/ipq40xx/files-6.12/arch/arm/boot/dts/qcom/qcom-ipq4029-insect-common.dtsi

```diff
@@ -330,27 +355,17 @@
 	};
 
 	i2c_0_pins: i2c_0_pinmux {
-		pinmux {
-			function = "blsp_i2c0";
-			pins = "gpio20", "gpio21";
-		};
-		pinconf {
-			pins = "gpio20", "gpio21";
-			drive-strength = <16>;
-			bias-disable;
-		};
+		function = "blsp_i2c0";
```

Comment:
Didn't this slip by in accidentally?
Edit: probably should go to a separate commit for alignment with `wired`-s